user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+
+.. hint::
+
+    By the user name, you can see the name of the business unit, that *"owns"* the user. This is also defined by the 
+    system administrator and serves as additional information for other users.
+
+**At least one business unit must be defined in the system.**
+
+Organizations
+^^^^^^^^^^^^^
+
+In addition to the business unit hierarchy, OroCRM Enterprise Edition provides the ability to create several 
+Organizations. Organizations may be described as large business units, independent enough in their operations to 
+require a high degree of separation and customization. Organizations provide a much higher degree of separation - every 
+organization in OroCRM is basically a business in its own right. They might have their own hierarchies of business 
+units, use different languages and locales, or even the most common entities such as Contacts or Accounts may differ 
+from one organization to another. Data records that exists in one organization are normally not available to another.
+
+Usually, organizations are need when their real-world counterparts have their own P&L reports and balance sheets. For 
+the telecom in the example above, different organization could represent offices in each of the states. Another good 
+example for several organizations would be a franchisor that has several franchisees - the best way to represent each 
+of them in OroCRM is a separate organization. Even a freelance entrepreneur working with several different businesses 
+from his computer will benefit using a separate organization to represent each of them in OroCRM.
+
+Subject to the work done in the company, some users may have access to several organizations 
+(work with multiple organization) or to all records across all organizations that are available to them 
+(work from a system organization). This is described below. 
+
+    
+.. _user-guide-getting-started-change-organization:
+
+Work with Multiple Organizations
+--------------------------------
+
+In many cases a user has access to a single organization. A sales representative in the U.S. office is likely to have no 
+access to the company’s French division.
+
+However, users can be assigned to multiple organizations just as they can be assigned to multiple business units and 
+will get access to the records in different organizations. For example, if you are working at the worldwide help desk 
+of a multinational organization, you should be able to access customer records in all the organization within your 
+company in order to open cases based on the customer requests. Similarly, if you are the CEO of a big enterprise 
+comprised of multiple independent divisions, you would like to have access to every one of them in order to check their 
+performance.
+
+As organizations are often much bigger and more separated than business units, the user will have to switch to the 
+necessary organization to work with data in each one of them.
```

Comment:
Edit: Since organizations are often much larger and more separated than business units, the user will have to switch to the necessary organization in order to work with data within each of them.